### PR TITLE
Add vue-bootstrap-typeahead by package.json

### PR DESCRIPTION
**Pull request by TheRealChrisS**
# Pull request template for new library

Pull request for issue: #
Related issue(s): # #

Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

## Profile of the lib

* Git repository (required): https://github.com/alexurquhart/vue-bootstrap-typeahead
* Official website (optional, not the repository): https://alexurquhart.github.io/vue-bootstrap-typeahead/
* NPM package url (optional): https://github.com/alexurquhart/vue-bootstrap-typeahead/blob/master/package.json
* License and its reference: https://github.com/alexurquhart/vue-bootstrap-typeahead/blob/master/LICENSE.txt
* GitHub / Bitbucket popularity (required):
  * Count of watchers: 3
  * Count of stars: 48
  * Count of forks: 33
* NPM download stats (optional):
  * Downloads in the last day: 297
  * Downloads in the last week: 1747
  * Downloads in the last month: 18063

## Essential checklist

* [ ] I'm the author of this library
  * [ ] I would like to add link to the page of this library on CDNJS on website / readme
* [x] This lib was not found on cdnjs repo
* [x] Doesn't already exist / has duplicate issue and PR
* [x] The lib has notable popularity
  * [ ] More than 200 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
  * [x] More than 800 downloads stats per month on npm registry
* [x] Project has public repository on famous online hosting platform (or been hosted on npm)

## Auto-update checklist

* [x] Has valid tags for each versions (for git auto-update)
* [x] Auto-update setup
* [x] Auto-update target/source is valid.
* [x] Auto-update filemap is correct.

## Git commit checklist

* [x] The first line of commit message is less then 50 chars; clean, clear and easy to understand.
* [x] The parent of the commit(s) in the PR is not older than 3 days.
* [x] Pull request is sent from a non-master branch with a meaningful name.
* [x] Separate unrelated changes into different commits.
* [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
* [x] Close corresponding issue in commit message
* [x] Mention related issue(s), people in commit message, comment.

### CODE REVIEW

File: ajax/libs/vue-bootstrap-typeahead/package.json [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+    "bootstrap",
+    "bootstrap 4",
+    "typeahead"
+  ],
+  "npmName": "vue-bootstrap-typeahead",
+  "npmFileMap": [
+    {
+      "basePath": "dist",
+      "files": [
+        "**/*"
```
Comment (extend1994) [inline ajax/libs/vue-bootstrap-typeahead/package.json]: Let's use `**/!(*.common.*|*.html)` to exclude `common.js` and `.html` files.

**Comment by extend1994** [comment]
Hi @TheRealChrisS 
I planned to help squash the commits and import versions but found the PR state was "wants to merge 2 commits into `cdnjs:master` from `unknown repository`" Could you check what's wrong? Thanks.

**Comment by PeterDaveHello** [comment]
@extend1994 Unknown repository means the forked repository has been deleted 😥

**Comment by extend1994** [comment]
@PeterDaveHello I once thought of it. Meanwhile, I saw https://github.com/TheRealChrisS/cdnjs/tree/add-vue-bootstrap-typeahead. So I think it would be better to check this issue with @TheRealChrisS 🙂

**Comment by PeterDaveHello** [comment]
@extend1994 Once the forked repo was deleted, the result can be undone, what you saw should be a newly forked one :)

**Comment by PeterDaveHello** [comment]
@TheRealChrisS as we didn't receive any response yet, I'll close this PR, but please feel free to open a new one, thanks for your contribution :+1:

**Pull request closed without merging**